### PR TITLE
[WIP] Fixes #11742 - Added a desc on the new settings additions

### PR DESCRIPTION
**Pull request by alongoldboim**

### CODE REVIEW

File: plugins/foreman_discovery/4.0/index.md [hunk 1]
```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The new discovery view uses regular expressions to decide which category each fact belongs to,
+its possible to change the regular expression used for each category by changing the values of
+discovery_facts_<category_name>
```
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: Angle brackets will be interpreted as HTML, escape them.

File: plugins/foreman_discovery/4.0/index.md [hunk 2]
```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The new discovery view uses regular expressions to decide which category each fact belongs to,
```
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: line width looks off

**Comment by domcleal** [comment]
Is this a Discovery 4.0 feature?  I don't see it.

**Comment by alongoldboim** [comment]
It will get in soon. https://github.com/theforeman/foreman_discovery/pull/200.
And fixing.

**Comment by domcleal** [comment]
Then you probably don't want to open a PR against the 4.0 docs, unless it's available in a point release yet.  I'll defer to the Discovery maintainers though, it's their documentation.

File: plugins/foreman_discovery/4.0/index.md (hunk 2 above)
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: I'd also suggest not referring to something as "new" in documentation, as it's presumably going to live on.

**Comment by alongoldboim** [comment]
@lzap asked me to add  it.
Anyway don't add it yet, ill notify on the PR when it gets in.
Im leaving to a PTO tomorrow so i just wanted it to be available when it gets in. 
Thanks.

File: plugins/foreman_discovery/4.0/index.md [hunk 3]
```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The discovery view uses regular expressions to decide which category each
+fact belongs to,
+its possible to change the regular expression used for each category by changing
```
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: it's

File: plugins/foreman_discovery/4.0/index.md [hunk 4]
```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The discovery view uses regular expressions to decide which category each
+fact belongs to,
```
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: line wrap is still messy

**Comment by alongoldboim** [comment]
@domcleal Better now?

File: plugins/foreman_discovery/4.0/index.md [hunk 5]
```diff
@@ -446,6 +446,11 @@ that, set the value of this setting to name of a fact reported. To hide the
 new column, set to a blank value. To show multiple columns, separate fact
 names by comma.
 
+The discovery view uses regular expressions to decide which category
+each fact belongs to, its possible to change the regular expression used for
```
Comment (domcleal) [inline plugins/foreman_discovery/4.0/index.md]: `it's` still

**Comment by lzap** [comment]
Please copy the `plugins/foreman_discovery/4.0/index.md` file to `plugins/foreman_discovery/5.0/index.md` and make the change there. Do not change anything else like index page and at the top of the page, please add a warning: **This is a documentation of upcoming release of plugin available in our nightly builds. Visit latest stable documentation instead.** With a link to our plugins index page (we can't actually link exactly on the correct plugin with this technique).

Please implement it as a yellow warning box we have for our old pages: http://theforeman.org/manuals/1.8/ (the top-left corner). Follow the same style defined in `_layouts/manual.html` and in the `_layouts/plugin.html` use a page variable called "warning" with two values of "old" or "unreleased" to render the required text (make your own for the former case).

This way we can add header to all our versions:

```
---
layout: plugin
title: Foreman Discovery 4.0 Manual
version: 3.0
warning: old
---
```

Add for all old versions and the "unreleased" for 5.0. Does it make sense?

**Comment by lzap** [comment]
The text is fine, I'd appreciate to see one simplified example of such a regexp. Pull out two facts into the generic section with that example please. Thanks for docs!

**Comment by alongoldboim** [comment]
On a PTO will be done on my return (7.10), good day.

**Comment by lzap** [comment]
I've implemented above in a separate PR, just keep this opened, I will merge once we merge the PR into develop. https://github.com/theforeman/theforeman.org/pull/457

**Pull request closed without merging**